tests/tests_e3/anod/test_sandbox.py

```diff
@@ -45,3 +48,189 @@ def test_sandbox_env():
     sandbox = e3.anod.sandbox.SandBox()
     sandbox.set_default_env()
     assert os.environ['GPR_PROJECT_PATH'] == ''
+
+
+def test_SandBoxCreate_git():
+    """Check if sandbox create can load the specs from a git repo"""
+    sandbox_dir = os.getcwd()
+    specs_dir = os.path.join(os.path.dirname(__file__), 'specs')
+    # prepare git repo
+    prepare_git(specs_dir)
+    with_git = e3.os.process.Run(
+        ['e3-sandbox', '-v',
+         'create',
+         '--spec-git-url', specs_dir,
+         sandbox_dir], output=None)
+    assert with_git.status == 0
+    # Test structure
+    for test_dir in ['bin', 'log', 'meta', 'patch',
+                     'specs', 'src', 'tmp', 'vcs']:
+                    assert os.path.isdir(test_dir)
+    # Test specs files if created
+    # Unless style checking is fixed prolog.py is not checked
```

Comment:
I don't understand? Why can't you generate the prolog.py when creating the git repository content?